data/configs/default.json

```diff
@@ -354,6 +354,51 @@
           }
         }
       }
+    },
+    {
+      "id": "RTSPtoWeb",
+      "properties": {
```

Comment:
Do not commit personal updates in your configuration file, especially if they are non standard (eg. require a third-party tool).